app/controllers/chargeback_controller.rb

```diff
@@ -247,6 +248,9 @@ def cb_rate_edit
                   detail.group = r[:group]
                   detail.per_unit = r[:per_unit]
                   detail.metric = r[:metric]
+                  # Copy the measure id of the rate_detail linkig with the rate_detail_measure
+                  id_measure = ChargebackRateDetailMeasure.find_by(:name => r[:measure]).id
+                  detail.chargeback_rate_detail_measure_id = r[:measure] ? id_measure : nil
```

Comment:
This line implies we won't...that is, `r[:measure]` could be nil, so the previous line would have blown up in that case.  I think a few specs around this might sort it out.  @dclarizio Can you or someone on your team help with the specs here, because I'm not sure how they are written in this part of the code.